README.md

```diff
@@ -59,6 +64,9 @@ To check that all the links in the documentation set are valid:
 
 3. Run `make build` or `make serve`. Broken links will be listed at the end of the build process.
 
+4. To render locally with link checking, you can instead use the command line: `ENABLED_HTMLPROOFER=true mkdocs serve`.
+This will add the 5 minute delay for URL validation to the server startup.
+
```

Comment:
Out of scope. Better to do this in another PR so the change log isn't polluted with unrelated content (the UCR points to this PR for change history).

Reply:
OK, I'll remove it.